packages/assistify-ai/client/views/app/tabbar/smarti.js

```diff
@@ -112,10 +112,58 @@ Template.AssistifySmarti.helpers({
 		} else if (instance.currentTryLoading.get() === instance.maxTriesLoading) {
 			return TAPi18n.__('Widget_could_not_load');
 		}
+	},
+	notArchived() {
+		return true;
```

Comment:
Don't we have archived channels?

Reply:
Also, the permission `archive-room` has to be validated. The semantic of the helper function should be `canArchiveRoom`

Reply:
I'm still searching for a way to check if a room is archived without checking the database.
Which would be a big performance impact every time you open the smarti widget.

Reply:
The `ChatRooms`-collection on the frontend is a cached collection. I'm not 100% sure that the archived-property is part of that though.
Still, accessing the database asynchronously, should not be an issue.